src/uvtheory/eos/hard_sphere_wca.rs

```diff
@@ -19,6 +19,23 @@ lazy_static! {
         [-2.905719617, -1.778798984, -1.556827067, -4.308085347],
         [0.429154871, 20.765871545, 9.341250676, -33.787719418],
     ]);
+
+//  New Fit by Anja to numerical integrals for uv-B3-theory
+    // #Tmin = 0.0, eta_max = 0.52, simultaneous fit
+
+    pub static ref WCA_CONSTANTS_ETA_A_UVB3: Array2<f64> = arr2(&[
+        [2.64043218,   -1.2184421 ,  -22.90786387,    0.96433414],
+     [-16.75643936,   30.83929771,   73.08711814, -166.57701616],
```

Comment:
or make it a constant like the other constants?

Reply:
I think Philipp was thinking of removing `lazy_static` and using static rust arrays instead of `ndarray`. For an example, see [the implementation of polar contributions}(https://github.com/feos-org/feos/blob/main/src/pcsaft/eos/polar.rs#L13-L19).

Reply:
And I'd also remove the `CONSTANTS` part of the names. Makes it a bit more concise and readable and the type system already makes it clear that this is a constant.

Reply:
To elaborate - `lazy_static` is used when you need multidimensional arrays (using `ndarrays`) where you want to have mathematical operations (such as matrix multiplication) in a more convenient manner. With native rust arrays, you can only use loops and indexing to perform element-wise operations. But since you already do that, static rust arrays are fine here.